tests/src/lib.rs

```diff
@@ -0,0 +1,72 @@
+use anyhow::Result;
+use common::{random_payload, send_get, send_post, OrderCreatedResponse, OrderStatusResponse};
+use std::time;
+mod common;
+
+const RETRY_TIMES: u32 = 5;
+const INTERVAL_IN_SECS: u64 = 5;
+const ORDER_STATUS_CREATED: &str= "created";
+const ORDER_STATUS_FULFILLED: &str = "fulfilled";
+
+#[tokio::test]
+async fn create_order_test() -> Result<()> {
+    let mut i = 0;
```

Comment:
nit: could `i` be `retry_count` instead for readability?

Reply:
oh yes